### PR TITLE
trader.js

### PR DESCRIPTION
**Pull request by Twttsunami**
Correcting the translation and specifying the words n_n

### CODE REVIEW

File: loc/npcs/trader.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ export default {
     bg: 'Продайте предметите си',
     cs: 'Prodávejte své zboží',
     pl: 'Sprzedawaj swoje przedmioty',
-    es: 'Vende sus ítemes',
+    es: 'Vende tus articulos',
```
Comment (dekdevy) [inline loc/npcs/trader.js]: is articulos better than itemes her? what are the meanings?

Comment (drifted-mdl) [reply loc/npcs/trader.js]: I pointed this out yesterday regarding a different file, there is no word for "item" in Spanish. "articulos" is the closest you can get. Verified by two people.

Comment (Twttsunami) [reply loc/npcs/trader.js]: hmmm dek devy sincerelly i don't know i pushed in translate google and it says items = Articulos :uuuuu

Comment (Twttsunami) [reply loc/npcs/trader.js]: dek ''Articulos'' in english means Articles and ''items'' not have means

Comment (Twttsunami) [reply loc/npcs/trader.js]: So i don't know means ''items''

Comment (Twttsunami) [reply loc/npcs/trader.js]: items maybe means ''Conjunto''
.-.

Comment (Twttsunami) [reply loc/npcs/trader.js]: or item's maybe too means ''Elementos''

Comment (Twttsunami) [reply loc/npcs/trader.js]: I think what items means ''elementos''

**Pull request merged**